app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -110,6 +111,11 @@ class PocketVideoFetchScheduler(
         return userFetchTime.timeInMillis - now.timeInMillis
     }
 
+    fun setDelayForQA(seconds: Long) {
+        IS_QA_BUILD = true
+        DELAY_TIME_FOR_QA_MILLIS = TimeUnit.SECONDS.toMillis(seconds)
```

Comment:
nit: -> qaFetchDelayOverrideMillis

I changed the word order to be consistent with the command line flag.

Related nit: ALL_CAPS is only used for constants, not for static state